core/keyboard_nav/cursor_svg.js

```diff
@@ -26,25 +26,20 @@
 
 goog.provide('Blockly.CursorSvg');
 
-goog.require('Blockly.Cursor');
-
-
 /**
  * Class for a cursor.
- * @param {!Blockly.Workspace} workspace The workspace to sit in.
- * @param {boolean=} opt_marker True if the cursor is a marker. A marker is used
- *     to save a location and is an immovable cursor. False or undefined if the
- *     cursor is not a marker.
- * @extends {Blockly.Cursor}
+ * @param {!Blockly.Workspace} workspace The workspace the cursor belongs to.
+ * @param {boolean} isMarker True if we want to use the marker rendering, false
+ *     otherwise. The marker rendering is a solid blue line instead of a flashing
+ *     red one.
  * @constructor
  */
-Blockly.CursorSvg = function(workspace, opt_marker) {
-  Blockly.CursorSvg.superClass_.constructor.call(this, opt_marker);
+Blockly.CursorSvg = function(workspace, isMarker) {
   this.workspace_ = workspace;
   this.constants = new Blockly.blockRendering.ConstantProvider();
   this.constants.init();
+  this.isMarker_ = isMarker;
```

Comment:
This needs an annotation.  so does the workspace_ above it.

Reply:
Why just the workspace and marker and not the constants?

Reply:
Sure constants as well.